### DISCUSSION TITLE
Query template parts

### DISCUSSION THREAD

**Discussion by jinsley8** [wp-graphql/wp-graphql#3230] [Q&A]
Is there any way to directly query template parts from the new full-site editing template parts section?

**Comment by justlevine** [comment]
Not yet, sadly.

**Reply by jinsley8** [reply]
@justlevine, do you have any suggestions on best practices for querying blocks and header/footer? Build header/footer separately in the app and query only the content blocks per page/post?

I was using [wpgraphql blocks plugin](https://wordpress.org/plugins/wpgraphql-blocks/) that queries the whole page (header/content blocks/footer) together. Should I only query content blocks with a plugin like [wpgraphql content blocks](https://github.com/wpengine/wp-graphql-content-blocks) instead?

The new block navigation is not queryable in the graphql either so is everyone still using the old menu system that is exposed as "menus" in graphql?
